meinberlin/apps/cms/templates/meinberlin_cms/home_page.html

```diff
@@ -14,6 +14,11 @@
     {% include 'meinberlin_cms/includes/cms_breadcrumb.html' %}
 {% endblock %}
 
+{% block extra_js %}
+    <script type="text/javascript" src="{% static 'react_map_teaser.js' %}"></script>
+    {{ block.super }}
```

Comment:
maybe the block.super should go before the new script we are including, in case of any js overwrites? and usually scripts are going after content.

Reply:
I moved the super above and added defer, as this is the modern way instead of moving it to the end of the body (https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML#applying_css_and_javascript_to_html)

Reply:
I discussed it with @philli-m  and we should start using defer (or async) everyhwere